Common/Helpers/BlobStorageManager.cs

```diff
@@ -10,15 +10,15 @@ namespace Microsoft.Azure.Devices.Applications.RemoteMonitoring.Common.Helpers
     /// <summary>
     /// Helper methods, related to blob storage.
     /// </summary>
-    public class BlobStorageHelper : IBlobStorageHelper
+    public class BlobStorageManager : IBlobStorageManager
     {
-        private readonly string _connectionString;
-        private readonly string _containerName;
+        private readonly ICloudBlobContainerProvider _blobContainerProvider;
 
-        public BlobStorageHelper(string connectionString, string containerName)
+        public BlobStorageManager(string connectionString, string containerName)
         {
-            _connectionString = connectionString;
-            _containerName = containerName;
+            CloudStorageAccount storageAccount = CloudStorageAccount.Parse(connectionString);
+            CloudBlobClient blobClient = storageAccount.CreateCloudBlobClient();
+            _blobContainerProvider = new CloudBlobContainerProvider(blobClient, containerName);
```

Comment:
Same connect as in CloudTableProvider. Instead of new CloudBlobContainerProvider have it injected in as part of the constructor.